hail/src/main/scala/is/hail/annotations/Region.scala

```diff
@@ -355,7 +355,13 @@ object Region {
           i += 1
         }
         v.leaveTuple()
-      case t: ComplexPType =>
+      case t: PCanonicalInterval =>
+        visit(t.representation, off, v)
+      case t: PCanonicalLocus =>
+        visit(t.representation, off, v)
+      case t: PCanonicalCall =>
+        visit(t.representation, off, v)
+      case t: PCanonicalNDArray =>
```

Comment:
Should these not use the `visitInterval`, `visitLocus`, etc. methods you added to `PrettyVisitor` in `ValueVisitor.scala`?

Reply:
oops, meant to remove that stuff from ValueVisitor. We need new methods on the ptypes that load the java rep from the region rep, so this is a workaround for now.